wgengine/netstack/netstack.go

```diff
@@ -789,12 +788,13 @@ func (ns *Impl) shouldHandlePing(p *packet.Parsed) (_ netip.Addr, ok bool) {
 }
 
 func netaddrIPFromNetstackIP(s tcpip.Address) netip.Addr {
-	switch len(s) {
+	switch s.Len() {
 	case 4:
+		s := s.As4()
 		return netaddr.IPv4(s[0], s[1], s[2], s[3])
 	case 16:
 		var a [16]byte
-		copy(a[:], s)
+		copy(a[:], s.AsSlice())
```

Comment:
use As16 instead. That's exactly what we need, rather than copying to a slice of length 16.

Reply:
Done in https://github.com/tailscale/tailscale/pull/9252/commits/8368ba975c2e4b9a00c7a3971f7982f3d35c7f02